stingray/events.py

```diff
@@ -511,3 +515,46 @@ def apply_deadtime(self, deadtime, inplace=False, **kwargs):
             new_ev = [new_ev, retall]
 
         return new_ev
+
+    def change_mjdref(self, new_mjdref):
+        """Change the MJD reference time (MJDREF) of the light curve.
```

Comment:
I'm totally happy to have this in here, but I can't help but ask ... why? What's the use case for this?

Reply:
To sum all photons from simultaneous observations, or anything that has to do with two different missions. 
My initial use case was using XMM and NuSTAR data to calculate cross spectra. Then I realized I could put all photons together for a large part of the observation and gain some sensitivity. Then I realized that we had this method for light curves and not for events :D